extensions/interactions/ItemSelectionInput/ItemSelectionInput.py

```diff
@@ -28,14 +28,23 @@ class ItemSelectionInput(base.BaseInteraction):
     answer_type = 'SetOfHtmlString'
 
     _customization_arg_specs = [{
-        'name': 'selection_count',
-        'description': 'Number of selections the student can make',
+        'name': 'max_allowable_selection_count',
+        'description': 'Maximum number of selections the student may make, '
+                       'please note that this is not the correct number of answers.',
```

Comment:
I don't think you need the second part.

"Maximum number of selections permitted" may be shorter.